ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml

```diff
@@ -71,8 +72,7 @@ metadata:
               "vipNetworkList": [
                 {
                   "networkName": "",
-                  "cidr": "",
-                  "v6cidr": ""
+                  "cidr": ""
```

Comment:
`v6cidr` is removed

Reply:
@akshayhavile  I have removed this intentionally, because ideally we will be specifying only one of these two.

Reply:
will customer get an idea that there is another option available if we specify only one?

Reply:
Have added the **v6cidr** field back.